setup.py

```diff
@@ -27,7 +27,9 @@ def readme():
     'klein == 15.3.1',
     'requests',
     'treq',
-    'Twisted',
+    # Twisted 17.1.0 causes problems with treq.testing
+    # https://github.com/twisted/treq/issues/164
+    'Twisted != 17.1.0',
```

Comment:
The [twisted ticket](http://twistedmatrix.com/trac/ticket/9032) suggests next versions to fall back to previous behaviour and drop support in 18.1 so skipping 17.1.0 should be OK.